utils.py

```diff
@@ -0,0 +1,19 @@
+import logging.config
+import os
+import yaml
+
+
+def setup_logging(default_path='config/logging.yaml',
+                  default_level=logging.INFO,
+                  env_key='LOG_CFG'):
+
+    path = default_path
+    value = os.getenv(env_key, None)
```

Comment:
Probably better:

```python
path = os.getenv(env_key, path)
```

Eliminates the need for a couple of assignments and a conditional.